src/components/CompareResults/RevisionRowExpandable.tsx

```diff
@@ -71,6 +72,26 @@ function RevisionRowExpandable(props: RevisionRowExpandableProps) {
   const newValues =
     newRunsReplicates && newRunsReplicates.length ? newRunsReplicates : newRuns;
 
+  const mode = useAppSelector((state) => state.theme.mode);
+
+  function getStyles(theme: string) {
+    const backgroundColor =
```

Comment:
Copying my comment from the other PR: 

This doesn't seem right to me. We have the light/dark mode setup here: https://github.com/mozilla/perfcompare/blob/main/src/theme/protocolTheme.ts#L10-L80

Here's an example of how it's used: https://github.com/mozilla/perfcompare/blob/0e215ca6936e5942403944fefb22a0b85e3ab0c3/src/components/CompareResults/TableHeader.tsx#L123